### DISCUSSION TITLE
GH API Get a release expects integer path variable

### DISCUSSION THREAD

**Discussion by jeremyfiel** [community/community#10385] [Other Feature Feedback, Questions, & Ideas] [answered]
Where can I find the `integer` for the `release_id`?  I'm only seeing the `tag` when viewing the `Releases` panel.
 
the URL is also pointing to the tag rather than the release number.  I tried to use below example but it's returning 404. 

https://docs.github.com/en/rest/reference/releases#get-a-release
<html>
<body>
<h4 id="get-a-release--parameters" style="box-sizing: border-box; margin-top: 24px; margin-bottom: 16px; font-weight: 600; font-size: 16px; line-height: 1.25; padding-top: 1rem; color: rgb(36, 41, 47); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><a href="https://docs.github.com/en/rest/reference/releases#get-a-release--parameters" style="box-sizing: border-box; background-color: transparent; color: var(--color-accent-fg); text-decoration: none;">Parameters</a></h4>

Name | Type | In | Description
-- | -- | -- | --
accept | string | header | Setting to application/vnd.github.v3+json is recommended.
owner | string | path |  
repo | string | path |  
release_id | integer | path | release_id parameter

</body>
</html>

### Example 
`https://api.github.com/repos/Redocly/redoc/releases/59`  returns 404

![image](https://user-images.githubusercontent.com/32110157/150425445-bda7f4dc-b2f3-427c-89f1-6724c7e16a6f.png)

```sh
GET https://api.github.com/repos/Redocly/redoc/releases/59
Authorization: <personal-access-token>
Accept: application/vnd.github.v3+json
```

**Comment by AlfieBrooks** [comment, marked as answer]
You can use get release by tag name to retrieve the `release_id`:
https://docs.github.com/en/rest/reference/releases#get-a-release-by-tag-name

**Comment by jeremyfiel** [comment]
Well that's a little silly.  the path parameter `release_id` doesn't even match the `Response` key of `id`

thanks @AlfieBrooks

**Reply by AlfieBrooks** [reply]
Yeah agreed, it's a bit annoying. That `id` is your `release_id` 👍
